client/app/components/project/package-section.hbs

```diff
@@ -0,0 +1,92 @@
+<div ...attributes>
+  <h2>{{@packageType}}</h2>
+
+  <div class="grid-x grid-padding-x">
+    <div class="cell large-6">
+      <ol class="no-bullet">
+        {{#each @packages as |package|}}
+          <Project::PackageListItem
+            @package={{package}}
+          />
+        {{/each}}
+      </ol>
+    </div>
+    <div class="cell large-6">
+      {{#let (flatten (map-by 'invoices' @packages)) as |invoices|}}
```

Comment:
Hmm. Each package can have multiple invoices, but each invoice is in an array itself?

Reply:
Having a hard time visualizing the before/after of this data structure

Reply:
no, that's not what this says. each package has many invoices. this is mapping out invoices from package _s_, plural.

Reply:
@allthesignals  Hmm, I was under the assumption we should list invoices by package. Just realized we have a bit of discrepancy between our current wireframe design and the wording on acceptance criteria. Maybe it's fine though

Reply:
> list invoices by package

@godfreyyeung hmm, I'm not sure how what I said contradicts this? It does list invoices by package. As far as I understand, there can't be invoices for draft packages.

Reply:
Let me know if we should huddle on this @godfreyyeung